src/components/DatePicker.js

```diff
@@ -0,0 +1,1029 @@
+    }
+  }
+
+  /**
+   * The function will decrease current month in state. It will also call props.onMonthChange.
+   */
+  _decreaseMonth() {
+    let newMonth;
+    let newYear;
+    if (this.state.month === 0) {
+      newMonth = 11;
+      newYear = this.state.year - 1;
+    } else {
+      newMonth = this.state.month - 1;
+      newYear = this.state.year;
+    }
+    this.setState({
+      month: newMonth,
+      year: newYear
+    });
+    if (this.props.onMonthChange) {
+      this.props.onMonthChange(newMonth + 1);
+    }
+  }
+
+  /**
+   * The function will increase current month in state. It will also call props.onMonthChange.
+   */
+  _increaseMonth() {
+    let newMonth;
+    let newYear;
+    if (this.state.month === 11) {
+      newMonth = 0;
+      newYear = this.state.year + 1;
+    } else {
+      newMonth = this.state.month + 1;
+      newYear = this.state.year;
+    }
+    this.setState({
+      month: newMonth,
+      year: newYear
+    });
+    if (this.props.onMonthChange) {
+      this.props.onMonthChange(newMonth + 1);
+    }
+  }
+
+  /**
+   * Function will return jsx for rendering the nav bar for calendar.
+   * Depending on following rules it will apply styles to prevMonthNav and nextMonthNav:
+   * 1. If disabled hide navs
+   * 2. If active apply activeStyles
+   */
+  _renderNavBar() {
+    const navBarStyle = {
+      ...style.navBarStyle,
+      ...this.props.navBarStyle
+    };
+    const monthLblStyle = {
+      ...style.monthLblStyle,
+      ...this.props.monthLblStyle
+    };
+    let prevMonthNavStyle = {
+      ...style.prevMonthNavStyle,
+      ...this.props.prevMonthNavStyle
+    };
+    let nextMonthNavStyle = {
+      ...style.nextMonthNavStyle,
+      ...this.props.nextMonthNavStyle
+    };
+    if (this.state.isPrevMonthNavActive) {
+      prevMonthNavStyle = {
+        ...prevMonthNavStyle,
+        ...style.activePrevMonthNavStyle,
+        ...this.props.activePrevMonthNavStyle
+      };
+    } else if (this.state.isNextMonthNavActive) {
+      nextMonthNavStyle = {
+        ...nextMonthNavStyle,
+        ...style.activeNextMonthNavStyle,
+        ...this.props.activeNextMonthNavStyle
+      };
+    }
+
+    return (
+      <div style={ navBarStyle }
+           className={ this.props.navBarClassName }>
+           { !this.props.disabled ? <span onMouseDown={ this._onPrevMonthNavMouseDown.bind(this) }
+                onMouseUp={ this._onPrevMonthNavMouseUp.bind(this) }
+                onTouchStart={ this._onPrevMonthNavTouchStart.bind(this) }
+                onTouchEnd={ this._onPrevMonthNavTouchEnd.bind(this) }
+                style= { prevMonthNavStyle }
+                className={ unionClassNames(this.props.prevMonthNavClassName, this.pseudoStyleIds.prevMonthNavStyleId) }></span> : void 0}
+          <span style={ monthLblStyle }
+                className={ this.props.monthLblClassName }
+                role="heading"
+                id={ this.state.month + '-' + this.state.year }>
```

Comment:
Hey @nikgraf , this was for a WAI-ARIA recommendation : http://www.w3.org/TR/wai-aria-practices/#datepicker

"The current month has a label representing the month and year. It is advisable to use a role heading but is not essential. This "label" should have a unique ID."